src/odemis/acq/stream/_static.py

```diff
@@ -385,47 +428,87 @@ def _updateImage(self):
     def _onPoint(self, pos):
         self._shouldUpdateImage()
 
+    def _onPolarization(self, pos):
+        self._shouldUpdateImage()
+
     def _setBackground(self, data):
         """Called when the background is about to be changed"""
```

Comment:
Please indicate the type of `data` (None, DataArray, or list of DataArrays), and add a not that it only returns either None or a list of DataArrays.